aws-ts-eks-migrate-nodegroups/README.md

```diff
@@ -2,3 +2,7 @@
 
 Creates an EKS cluster with node groups and a workload, and showcases how add an
 additional node group to use for workload migration with zero downtime.
+
+For step-by-step instructions, check out the [full tutorial][tutorial-migrate-nodegroups].
+
+[tutorial-migrate-nodegroups]: {{< relref "/docs/reference/tutorials/kubernetes/tutorial-eks-migrate-nodegroups" >}}
```

Comment:
```suggestion
[tutorial-migrate-nodegroups]: https://www.pulumi.com/docs/reference/tutorials/kubernetes/tutorial-eks-migrate-nodegroups/
```

Need to specify the full URL here as we're not running this through Hugo.

Reply:
Oops, nice catch. Had `pulumi/docs` on the brain 😄